consensus/polybft/contractsapi/bindings-gen/main.go

```diff
@@ -447,7 +447,7 @@ import (
 		log.Fatal(err)
 	}
 
-	if err := ioutil.WriteFile("./consensus/polybft/contractsapi/contractsapi.go", output, 0600); err != nil {
+	if err = os.WriteFile("./consensus/polybft/contractsapi/contractsapi.go", output, 0600); err != nil {
```

Comment:
WDYT about using `common.SaveFileSafe` function?

```suggestion
	if err = common.SaveFileSafe("./consensus/polybft/contractsapi/contractsapi.go", output, 0600); err != nil {
```

Reply:
Done